api/v1alpha1/httproute_types.go

```diff
@@ -216,6 +220,20 @@ type HTTPRouteAction struct {
 	// "myroutetarget" in group "networking.acme.io").
 	ForwardTo *RouteActionTargetObjectReference `json:"forwardTo" protobuf:"bytes,1,opt,name=forwardTo"`
 
+	// ServicePort specifies the destination port to use when ForwardTo is
```

Comment:
I would rather have this use a more generic name, maybe something like DestPort or TargetPort? We want this to be generic enough to apply to resources that aren't Services.

Reply:
Would it make sense for this to be part of the `ForwardTo` reference?